deploy/helm/charts/README.md

```diff
@@ -33,7 +33,7 @@ Please visit the [link](https://openebs.github.io/dynamic-localpv-provisioner/)
 
 ```console
 # Helm
-$ helm install [RELEASE_NAME] openebs-localpv/localpv-provisioner
+$ helm install [RELEASE_NAME] openebs-localpv/localpv-provisioner --namespace [NAMESPACE]
```

Comment:
Why is the [command](https://github.com/openebs/dynamic-localpv-provisioner/blame/gh-pages/index.md#L28) specified in the `index.md` different from this.

Reply:
Please review the PR with the fix here: https://github.com/openebs/dynamic-localpv-provisioner/pull/51